scripts/stubtest/allowlist.txt

```diff
@@ -14,6 +14,10 @@ django.contrib.auth.migrations.*
 django.contrib.flatpages.migrations.*
 django.contrib.contenttypes.migrations.*
 
+# Annotate optional parameters which Django accepts as **kwargs
+django.contrib.gis.db.models.GeometryField.formfield
+django.contrib.gis.db.models.fields.GeometryField.formfield
+
```

Comment:
With the addition of `*,` to `GeometryField.formfield(...)`, these are no longer needed.

```suggestion
```